app/views/organizations/staff/dashboard/index.html.erb

```diff
@@ -46,6 +46,7 @@
         <% end %>
       </div>
     </div>
+    <%= render 'task'%>
     <%= turbo_frame_tag "tasks-frame" %>
```

Comment:
Do we need this turbo frame tag still?

Reply:
Yes, they are still in use, but I'm not very familiar with them. I need some time to understand the underlying functionality before integrating it with the current scenario. For now, it's working fine.

Reply:
If you look at the markup in the dev console it should just be `<turbo-frame id="tasks-frame">` because `    <%= turbo_frame_tag "tasks-frame" %>` is just a helper from rails to create the markup.

Reply:
I think you can delete the `<%= turbo_frame_tag "tasks-frame" %>` given you moved the same frame into the partial now?